core/http/src/lib.rs

```diff
@@ -20,6 +20,9 @@ pub mod hyper;
 pub mod uri;
 pub mod ext;
 
+#[doc(hidden)]
+pub mod unix;
```

Comment:
I think we still need `cfg(any(unix, windows))` here and everywhere else since `cfg(unix)` + `cfg(windows)` is not every platform.

Reply:
Fixed.